src/nvim/os/input.c

```diff
@@ -61,10 +61,11 @@ void input_start_stdin(int fd)
 void input_stop_stdin(void)
 {
   if (!read_stream) {
-    return;
+    input_start_stdin(0);
```

Comment:
This is a little hokey, but at the same time I'm trying to avoid having to re-invent a platform-agnostic wrapper around `fnctl`. libuv already does that for us (`uv_stream_set_blocking`), and we also have this `RStream` wrapper. Initializing either of those is quite a chore.